projects/step-frontend/src/lib/modules/timeseries/chart/tooltip-plugin.ts

```diff
@@ -15,8 +16,15 @@ interface Anchor {
   right?: number;
 }
 
+interface TooltipPluginSettings {
+  yScaleUnit?: string; // string to append on the y axis values
+  zAxisLabel?: string;
+}
+
 export class TooltipPlugin {
-  public static getInstance(yScaleUnit?: string): uPlot.Plugin {
+  static readonly SUMMARY_BARS_COLOR = '#97a7d7';
```

Comment:
We should have somewhere a colours modell containing all colours used in the visualization so it is easy to override them if we need to support different colour schemes

Reply:
Probably nothing unique to this PR, just a general observation

Reply:
for now I will move them in my time-series.config.ts file